pkg/cmd/gitprovider/add.go

```diff
@@ -29,10 +29,11 @@ var GitProviderAddCmd = &cobra.Command{
 		setGitProviderConfig := apiclient.SetGitProviderConfig{}
 		setGitProviderConfig.BaseApiUrl = new(string)
 		setGitProviderConfig.Username = new(string)
+		setGitProviderConfig.Alias = new(string)
 
-		gitprovider_view.GitProviderSelectionView(&setGitProviderConfig, nil, false)
+		gitprovider_view.GitProviderSelectionView(&setGitProviderConfig, nil, apiClient, ctx)
```

Comment:
![Screenshot 2024-09-23 at 09 49 40](https://github.com/user-attachments/assets/d4781345-e64d-4143-8a87-1c3d7c64cd28)

Where is all the text from the form? The text that shows the help and required scopes?